string node
string text
---
bool result
